workbench/src/renderer/menubar/report.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+import i18n from '../../shared/i18n';
+import { Translation } from 'react-i18next';
+import {
+  handleClickExternalURL,
+  handleClickFindLogfiles
+} from './handlers';
+import { getSettingsValue } from '../components/SettingsModal/SettingsStorage';
+
+const language = await getSettingsValue('language');
+await i18n.changeLanguage(language);
+ReactDom.render(
+  <Translation>
+    {(t, { i18n }) => (
+      <React.Fragment>
+      <p id="header">
+        {t('Please help us by reporting problems.')}
+      </p>
+      <p>
+        <b>{t('If the problem is related to a specific InVEST model, ')}</b>
+        {t('please see the guidelines here for reporting problems: ')}
+        <a
+          href="https://community.naturalcapitalproject.org/t/guidelines-for-posting-software-support-questions/24"
+        >
+          {t('Guidelines for posting software support questions')}
+        </a>
+      </p>
+      <p>
+        <b>{t('If the problem is related to this User Interface, ')}</b>
+        {t('rather than with a specific InVEST model,')}
```

Comment:
I hope it's okay to translate these two strings separately. I like the visual hierarchy the page has by bolding the first line, but it's clearly a tradeoff.

Reply:
Yeah, it's probably ok